expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -161,7 +161,7 @@ void shouldReturnIndexDocumentWithValidContributorAffiliationCountryCode() throw
         var indexDocument = fromPublication(fakeUriRetriever, resourceService, publication);
         var framedResultNode = indexDocument.asJsonNode();
 
-        var actualCountryCode = framedResultNode.at("/entityDescription/contributors/1/affiliations/0/countryCode")
+        var actualCountryCode = framedResultNode.at("/entityDescription/contributors/1/affiliations/0/country")
```

Comment:
We have changed this… @elkebab probably relevant for you.

Reply:
This requires updating a json pointer in `nva-nvi` as well. We should coordinate deployment

Reply:
Ok, will make it "countryCode"